core/geotime/src/main/java/mil/nga/giat/geowave/core/geotime/index/dimension/TemporalBinningStrategy.java

```diff
@@ -361,6 +361,30 @@ private Calendar getStartEpoch(
 		return cal;
 	}
 
+	/**
+	 * The == operator is not reliable for doubles, so we are using this method
+	 * to check if two doubles are equal
+	 * 
+	 * @param x
+	 * @param y
+	 * @return true if the double are equal, false if they are not
+	 */
+	private static boolean checkIfDoublesEqual(
+			double x,
+			double y ) {
+		boolean xNeg = false;
+		boolean yNeg = false;
+		double diff = (Math.abs(x) - Math.abs(y));
+
+		if (x < 0.0) {
+			xNeg = true;
+		}
+		if (y < 0.0) {
+			yNeg = true;
+		}
+		return (diff <= 0.0001 && diff >= -0.0001 && xNeg == yNeg);
```

Comment:
suggest a smaller epsilon than .0001?  ...the implication is minor, on trying to define a dateline crossing 179.99995 to -179.00005 would assume the whole world rather than across the dateline, but we should just be able to bump the epsilon to something much smaller